src/state/trajectory/types.ts

```diff
@@ -117,8 +111,9 @@ export interface HealthCheckTimeout {
     [requestId: string]: NodeJS.Timeout;
 }
 
-export type ConversionStatus =
-    | typeof CONVERSION_INACTIVE
-    | typeof CONVERSION_NO_SERVER
-    | typeof CONVERSION_SERVER_LIVE
-    | typeof CONVERSION_ACTIVE;
+export enum ConversionStatus {
+    Inactive = "CONVERSION_INACTIVE",
+    NoServer = "CONVERSION_NO_SERVER",
+    ServerLive = "CONVERSION_SERVER_LIVE",
```

Comment:
is this one necessary? it doesn't seem mutually exclusive with the others. Presumably the server is live while conversion in all cases other than "no server". I also don't see anything that uses `if CONVERSION_STATUS == ServerLive`

Reply:
When we go the conversion form we assume "no server" until the health check comes back, but conversion isn't "active" till we send the file, so "server live" is the in between state that where we are good to proceed. The way things are written now I believe we do need it.